File: desafio-02/leonelsanchesdasilva/go/desafio02.go
package main

import "fmt"
import "math"

func main() {
	for i := 1; i <= 10000; i++ {
		var primo bool = true;
		var raizQuadrada int = int(math.Floor(math.Sqrt(float64(i))));
		for j := 2; j <= raizQuadrada; j++ {
			if (i % j == 0) {
				primo = false;
				break;
			}
		}
		if (primo) {
			fmt.Println(i);
		}
	}
}

File: desafio-02/leonelsanchesdasilva/java/desafio02.java
class Desafio02 {
  public static void main(String[] args) {
    for (int i = 1; i <= 10000; i++) {
      Boolean primo = true;
      for (int j = 2; j <= Math.floor(Math.sqrt(i)); j++) {
        if (i % j == 0) {
          primo = false;
          break;
        }
      }

      if (primo) {
        System.out.println(i);
      }
    }
  }
}
